### PR TITLE
Re-arrange serialization tests

### PR DESCRIPTION
**Pull request by sadym-chromium**
Move the test of the property name to always be a string to serialization tests to unscreen preload implementation.

### CODE REVIEW

File: webdriver/tests/bidi/script/add_preload_script/arguments.py
Comment (mathiasbynens) [inline webdriver/tests/bidi/script/add_preload_script/arguments.py:24]: Why change this? We definitely need to support string values that look like numbers, so  I’m not sure why we’d avoid this in a test.

File: webdriver/tests/bidi/script/add_preload_script/arguments.py
Comment (mathiasbynens) [inline webdriver/tests/bidi/script/add_preload_script/arguments.py:75]: Same question here. Why? JavaScript property names are strings. It shouldn’t matter whether that string is `'42'` or `'qux'`, so what’s the motivation for changing this test?

**Review by mathiasbynens — CHANGES_REQUESTED** [review]

File: webdriver/tests/bidi/script/add_preload_script/arguments.py
Comment (sadym-chromium) [reply webdriver/tests/bidi/script/add_preload_script/arguments.py:24]: To unscreen the preload tests. I added the tests for serialization.

File: webdriver/tests/bidi/script/add_preload_script/arguments.py
Comment (sadym-chromium) [reply webdriver/tests/bidi/script/add_preload_script/arguments.py:75]: They are moved to dedicated serialization tests.

File: webdriver/tests/bidi/script/add_preload_script/arguments.py
Comment (mathiasbynens) [reply webdriver/tests/bidi/script/add_preload_script/arguments.py:24]: What does “unscreen” mean?

**Comment by mathiasbynens** [comment]
The changes to `webdriver/tests/bidi/script/call_function/result.py` and `webdriver/tests/bidi/script/evaluate/result.py` LGTM. I’m not convinced the other changes are necessary.

**Comment by sadym-chromium** [comment]
> In case some feature doesn't fully work yet cannot you just mark this specific test as expected fail? 

Yes, we have an issue in serializing object property names. If they can be parsed to int, they are considered to be number, not string: https://crrev.com/c/4604009.

> We have a couple of those scenarios as well and marked them all as such. We should not change valid tests only because one remote end cannot handle it correctly yet.

I don't insist on merging the change to the preload script, but it seemed making tests more atomic and testing only 1 feature.  I agree that to test preload scripts we need the serialization to be implemented. So I can remove the change in the preload tests, but still add them to serialization.

**Review by whimboo — APPROVED** [review]
Ok, so moving the specific failing tests for a number as key to the general serialization tests makes sense to me. We did the same for `log.entryAdded` which only contains a rough set of validations.

I really wonder why only the Safari wpt.fyi job has been run, which doesn't implement BiDi at all right now. :/

**Comment by whimboo** [comment]
@sadym-chromium I assume we are waiting for a review from @mathiasbynens here?

**Review by mathiasbynens — APPROVED** [review]

**Comment by whimboo** [comment]
@jgraham could you please admin merge? Thanks!

**Pull request merged**
